### DISCUSSION TITLE
Features display

### DISCUSSION THREAD

**Discussion by chunibyocola** [chunibyocola/sc-translator-crx#19] [Show and tell]
This page will display some features of ScTranslator.
这个页面会展示“可可翻译”的部分功能。
このページには、いくつかの「ココ翻訳」の機能が表示されます。

# My daily use / 私の日常的に使用 / 我的日常使用
![daily-use](https://user-images.githubusercontent.com/46666257/169047343-de19d71a-2fab-4cb2-8807-29da6e31038d.gif)

**Comment by chunibyocola** [comment]
# Translate history / 翻译历史 / 翻訳記録
![translate-history](https://user-images.githubusercontent.com/46666257/125593328-072b906e-12ac-43ed-985d-03ba915a7a92.gif)

**Comment by chunibyocola** [comment]
# "Call out extension in web page" keyboard shortcut　/ “在网页中唤出扩展”快捷键 / 「ウェブページで拡張機能を呼び出す」キーボードショートカット
![call-out-panel](https://user-images.githubusercontent.com/46666257/125598915-9a10e2ab-cee8-4289-bdfc-7ee7ee395766.gif)

**Comment by chunibyocola** [comment]
# Multiple Translate mode / 复数翻译模式 / 複数翻訳モード
![multiple-translate](https://user-images.githubusercontent.com/46666257/125599088-362c7aef-846a-4ed2-8c13-1a37db9de7b5.gif)

**Comment by chunibyocola** [comment]
# PDF Viewer / PDF 阅读器 / PDFビューア
![pdf-viewer](https://user-images.githubusercontent.com/46666257/125599266-89db0f8d-788f-40fb-b5d8-4bba47bb077f.gif)

**Comment by chunibyocola** [comment]
# Respond to separate translate window / 响应到独立翻译窗口 / スタンドアロン翻訳ウィンドウに応答する
![respond-to-separate](https://user-images.githubusercontent.com/46666257/125602086-b713ed1b-b76d-40ac-8712-94f48658d4c8.gif)

**Comment by chunibyocola** [comment]
# Separate translate window / 独立翻译窗口 / スタンドアロン翻訳ウィンドウ
![separate-window](https://user-images.githubusercontent.com/46666257/125602284-e94410dc-4a74-4d1d-9111-8f5c12fd2d5c.gif)

**Comment by chunibyocola** [comment]
# Switch theme / 切换主题 / テーマの切り替え
![switch-theme](https://user-images.githubusercontent.com/46666257/125602401-52b6ed2d-a307-4e28-8062-3dee8c7b0acb.gif)

**Reply by Wei-Winter** [reply]
这个可以跟随浏览器主题自动切换主题吗

**Comment by chunibyocola** [comment]
# Always inside the window / 保持在窗口内 / 窓内にキープ
![always-in-the-window](https://user-images.githubusercontent.com/46666257/125602577-8735bdda-8ec8-4ae6-bffa-750c24aa541c.gif)

**Comment by chunibyocola** [comment]
# Relevant to translate button / 翻译按钮相关 / 翻訳ボタンに関して
![translate-button-relevant](https://user-images.githubusercontent.com/46666257/125602789-f48764ed-8509-442a-9985-000b27794a92.gif)

**Comment by chunibyocola** [comment]
# Pin the panel while opening it / 打开面板时将其固定 / パネルを開くときに固定する
![pin-the-panel-while-opening-it](https://user-images.githubusercontent.com/46666257/125602949-48532a7b-2cb4-41b2-8c0f-0fd6cc22e63a.gif)

**Comment by chunibyocola** [comment]
# Remember the position of pinned panel / 记住面板固定的位置 / 固定されたパネルの位置を記憶する
![remember-the-pinned-panel-position](https://user-images.githubusercontent.com/46666257/125603034-1922079d-6752-4b44-8a1e-fa91275e9fe4.gif)

**Comment by chunibyocola** [comment]
# Translate directly while panel pinning / 翻译面板固定时直接翻译 / 翻訳パネルの固定中に直接翻訳する
![translate-directly-while-pinning](https://user-images.githubusercontent.com/46666257/125603158-5e477cc4-db90-4140-8af0-703d296e20cc.gif)

**Comment by chunibyocola** [comment]
# Sliders of panel's status / 面板状态的滑块 / パネルスターテスのスライダー
![panel-status-slider](https://user-images.githubusercontent.com/46666257/125603333-e1ae2a2e-2006-4890-ba09-721d81672149.gif)

**Comment by chunibyocola** [comment]
# Insert translate result / 插入翻译结果 / 翻訳結果を挿入
![insert-result](https://user-images.githubusercontent.com/46666257/125603923-a1b2368f-de37-496d-b6d6-ffee3f760cb9.gif)

**Comment by chunibyocola** [comment]
# Text preprocessing / 文本预处理 / テキストの前処理
![text-preprocessing](https://user-images.githubusercontent.com/46666257/125604015-35678402-47ad-4e97-95e2-6a9935b3d68e.gif)

**Comment by chunibyocola** [comment]
# Web page translating / 网页翻译 / Webページ翻訳
![web-page-translating](https://user-images.githubusercontent.com/46666257/127119856-609d20c4-5e78-42f7-a5ad-7a49370caf0e.gif)
